rds-instance-full/main.tf

```diff
@@ -40,6 +40,29 @@ module "rds_security_group" {
   ingress_rules       = ["${var.ingress_rule}"]
 }
 
+module "custom_security_group" {
+  source = "terraform-aws-modules/security-group/aws"
+
+  create = "${var.rds_sg == "" ? 0 : 1 }"
+
+  name        = "${local.identifier}-rds-ecs"
+  description = "Security group with RDS ports open for ECS"
+  vpc_id      = "${data.terraform_remote_state.vpc.vpc_id}"
+
+  ingress_cidr_blocks = ["${data.terraform_remote_state.vpc.vpc_cidr_block}"]
+
+  ingress_with_source_security_group_id = [
+    {
+      rule                     = "${var.ingress_rule}"
+      source_security_group_id = "${var.rds_sg}"
+    },
+  ]
+}
+
+locals {
+  security_group_id = "${coalesce(join("", module.custom_security_group.*.security_group_id), module.rds_security_group.this_security_group_id)}"
```

Comment:
If I remember correctly, `join("", module.custom_security_group.*.security_group_id)` can be replaced with `module.custom_security_group.security_group_id`.

Reply:
No, it didn't work when I was working on it as addition for the RDS module.

Reply:
@antonbabenko I will test this in RDS Neo module anyway, so I can do new PR removing this if it wouldn't be necessary